### ISSUE TITLE
IndieAuth login fails with WordPress plugin

### ISSUE THREAD

**Issue by snarfed** [marksuth/indigenous-android#430]
Hi @marksuth! When I try to add an IndieAuth account for snarfed.org, which is running v4.2.1 (ie the latest) of the IndieAuth WordPress plugin, I get this error in the in-app browser:

```json
{"error":"rest_invalid_param","error_description":"Invalid parameter(s): me","params":{"me":"Invalid parameter."},"details":[]}
```

@dshanske is this an incompatibility with the full authorization_code grant vs the IndieAuth-only profile URL response?

**Comment by marksuth** [comment]
I've tested this morning with a test WordPress site I've got hidden away and was able to add an account successfully. Are you installing from F-Droid by any chance?

**Comment by snarfed** [comment]
I installed from the Play Store. Were you using HEAD here instead? Or maybe an older version of the WP plugin?

**Comment by snarfed** [comment]
Looks like this is fixed in the new IndiePass v0.81 release! 🎉
